librz/core/cmd/cmd_help.c

```diff
@@ -430,6 +430,104 @@ RZ_IPI void rz_core_clippy_print(RzCore *core, const char *msg) {
 	}
 }
 
+RZ_API void rz_core_cmd_help_calc_expr(RzCore *core, const char *input) {
```

Comment:
Since you are modifying this code, I suggest porting this particular command to the rzshell as well, and use `RzCmdStateOutput`, just add corresponding code into `librz/core/cmd_descs/cmd_help.yaml`. And extract evaluation function to the API, while keeping printing function as the IPI.